CODE_OF_CONDUCT.md

```diff
@@ -1,4 +1,4 @@
-# Contributor Covenant Code of Conduct
+# Community Code of Conduct
```

Comment:
Maybe it's better to simplify it more to `Code of Conduct`?) I think it's a popular subject and everyone will understand that it's about community.

Reply:
@ovr, yes I also thought about that option but I think adding the word "community" makes it sound more friendly. If you just have `Code of Conduct` at the top of the page, I was afraid it may look too much like a legal document 😃  WDYT?